obspy/signal/spectral_estimation.py

```diff
@@ -401,7 +405,7 @@ def __init__(self, stats, metadata, skip_on_gaps=False,
 
         # save things related to kwargs
         self.skip_on_gaps = skip_on_gaps
-        self.db_bins = db_bins
```

Comment:
Why that deletion?

Reply:
Yikes - this was an inadvertent deletion. Thanks for catching that. I will add it back in.

Reply:
Thanks!